constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
```

Comment:
I think `prefix` is correct, but since the prefixes are for grouping labels, maybe something like:

```suggestion
const LABEL_GROUPS = {
```